src/backend/Dockerfile.pangolin

```diff
@@ -40,18 +40,17 @@ RUN apt-get update && apt-get install -y  \
 
 # Install Pangolin and PangoLEARN + deps (for ncov)
 # The cov-lineages projects aren't available on PyPI, so install via git URLs.
-RUN pip3 install snakemake==6.8.0
+RUN pip3 install snakemake==7.30.1
 RUN pip3 install git+https://github.com/cov-lineages/pangolin.git@v4.3
 RUN pip3 install git+https://github.com/cov-lineages/pangolin-data.git@v1.21
 RUN pip3 install git+https://github.com/cov-lineages/scorpio.git@v0.3.17
 RUN pip3 install git+https://github.com/cov-lineages/constellations.git@v0.1.12
-RUN pip3 install pysam==0.19.1
```

Comment:
this seems to come from somewhere else 🤔 Without this line it's still installed into python3.10 folder